target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -33,6 +33,11 @@ netgear,r7800)
 	ucidef_set_led_netdev "wan" "WAN" "${boardname}:white:wan" "eth0"
 	ucidef_set_led_ide "esata" "eSATA" "${boardname}:white:esata"
 	;;
+qxwlan,e5200)
+	ucidef_set_led_usbport "usb1" "USB 1" "${boardname}:green:usb1" "usb1-port1" "usb2-port1"
+	ucidef_set_led_usbport "usb2" "USB 2" "${boardname}:green:usb3" "usb3-port1" "usb4-port1"
+	ucidef_set_led_ide "esata" "eSATA" "${boardname}:green:esata"
```

Comment:
(see below, this can be dropped with [disk-activity](https://github.com/openwrt/openwrt/pull/1935/commits/d4e5600c9c0583fd3a37d78350dd32c177605f81#r266672399) device-tree trigger ).